copter/source/docs/autotune.rst

```diff
@@ -83,7 +81,7 @@ If the vehicle feels sloppy after the AutoTune, try increasing the :ref:`AUTOTUN
 Invoke AutoTune with Position Hold
 ==================================
 
-In Copter-3.5 (and higher) AutoTune performs a weak position hold if invoked from Loiter or PosHold flight modes (as opposed to AltHold).
+In Copter-3.5 (and higher) AutoTune performs a weak position hold if invoked from Loiter or PosHold flight modes (as opposed to AltHold) while doing a pitch or roll autotune (not yaw).
```

Comment:
are you sure it doesn't do it in yaw? It should.I've just tested and it does do weak pos hold in yaw autotune

Reply:
at this point no....fixing...